src/main/java/ai/rev/helpers/ErrorInterceptor.java

```diff
@@ -31,6 +32,8 @@ public Response intercept(Interceptor.Chain chain) throws IOException {
           throw new AuthorizationException(errorResponse);
         case 400:
           throw new InvalidParameterException(errorResponse);
+        case 403:
```

Comment:
Maybe we should adjust the default case so we can capture any future error codes we might want to adopt without having to update the SDKs?

Reply:
Changing the string format should make it so the error code and response object show by default